gradle.properties

```
@@ -1,6 +1,7 @@
 projectName='blockingqueue'
 packageOrg=ayash
 group=io.ayash
-version=1.0-SNAPSHOT
-ballerinaLangVersion=2201.10.2
+version=1.0.0-SNAPSHOT
```

Since this is an initial version lets use `0.1.0`.
```suggestion
version=0.1.0-SNAPSHOT
```